QUALITY_DECLARATION.md

```diff
@@ -125,19 +126,32 @@ Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linu
 
 ## Dependencies [5]
 
-`class_loader` has run-time and build-time dependencies that need to be considered for this declaration.
+Below are evaluations of each of `class_loader`'s run-time and build-time dependencies that have been determined to influence the quality.
+
+It has one "buildtool" dependency, which do not affect the resulting quality of the package, because they do not contribute to the public library API.
+It also has several test dependencies, which do not affect the resulting quality of the package, because they are only used to build and run the test code.
 
 ### Direct Runtime ROS Dependencies [5.i]/[5.ii]
 
-- console_bridge_vendor
-- rcpputils
+#### `console_bridge_vendor`
+
+The `console_bridge_vendor` package provides a wrapper around console_bridge, providing nothing but a dependency on console_bridge, on some systems. On others, it provides an ExternalProject build of console_bridge.
+
+It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/console_bridge_vendor/blob/master/QUALITY_DECLARATION.md).
```

Comment:
@ahcorde meta: should we re-state all dependencies' quality level for each package, as opposed to just linking to the quality declarations? Won't that result in a maintenance nightmare?

Reply:
Somehow we need to indicate the Quality Level of the other packages, because the Quality Level of this package depends on the others. This is how we have decided to go ahead, but we are open to suggestions. +1 about the nightmare

Reply:
Hmm, what about having an standard icon or badge for each QL that packages can keep next to their QDs? Like a certification seal, that can be included by the QD of the package itself and the QDs of the packages that directly depend on it.

It won't auto-update recursively when something in the dependency graph changes though.

Reply:
As discussed during the meeting, let's not block this QD on this issue.